index.js

```diff
@@ -58,7 +58,7 @@ class SnackbarComponent extends Component {
           }}
         >
           <Text style={[styles.text_msg, {color: this.props.messageColor}]}>{this.props.textMessage}</Text>
-          {this.props.actionHandler && this.props.actionText &&
+          {this.props.actionHandler && !!this.props.actionText &&
```

Comment:
```suggestion
          {this.props.actionHandler !== noop && !!this.props.actionText &&
```
Also seems like a good idea, otherwise only having the actionText would show the action button.
What do you think?